website/docs/reference/resource-configs/trino-configs.md

```diff
@@ -0,0 +1,335 @@
+---
+title: "Starburst (Trino) configurations"
+id: "trino-configs"
+---
+
+
+## Model-level configuration
+
+### Session properties
+
+In some specific cases, there may be needed tuning through the Trino session properties only for a specific dbt model. In such cases, using the [dbt hooks](https://docs.getdbt.com/reference/resource-configs/pre-hook-post-hook) may come to the rescue:
+
+```sql
+{{
+  config(
+    pre_hook="set session query_max_run_time='10m'"
+  )
+}}
+```
+
+### Use table properties to configure connector specifics
+
+Trino connectors use table properties to configure connector specifics.
+
+Check the Trino connector documentation for more information.
```

Comment:
As someone who doesn't know trino - would I be correct that the user needs to check out the specific Trino connector for context (and is this in the context of writing using the connector?).

Reply:
That's correct, connectors differ from each other, including writing support.